zgw_consumers/models/services.py

```diff
@@ -113,6 +132,24 @@ def clean(self):
                 {"header_key": _("If header_value is set, header_key must also be set")}
             )
 
+    @property
+    def connection_check(self) -> str | None:
```

Comment:
type annotations are still wrong - `Response.status_code` is type `int`, not `str`.

```suggestion
    def connection_check(self) -> int | None:
```